cmd/migrate/main.go

```diff
@@ -16,6 +16,8 @@ var (
 )
 
 func main() {
+	//initialise config
+	config.BuildConfig()
```

Comment:
build config after args are validated. If invalid args are passed then do not need to read config.

Reply:
this is handled inside the config function.